recipes/libertem-holo/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "libertem-holo" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/libertem_holo-{{ version }}.tar.gz
+  sha256: 1f9e8025a48a5106834feea69edb67e573b566a73a4f670e117405391667abf1
+
+build:
+  skip: true  # [py<39]
```

Comment:
This one can probably be noarch, right?

Reply:
Ah, I think it's because the original packaging had some more constrained version to nudge pip/uv, which I removed for conda; as I understand it, that would prevent `noarch`.